src/error.rs

```diff
@@ -75,6 +80,18 @@ impl IntoResponse for HcHttpGatewayError {
                 StatusCode::BAD_GATEWAY,
                 Json(ErrorResponse::from("Could not connect to Holochain")),
             ),
+            HcHttpGatewayError::AppSelectionError(AppSelectionError::NotInstalled) => (
```

Comment:
I find the app selection errors are overrepresented.

Reply:
Responded above, I'm not sure I've got a clear picture of what to do about this one

Reply:
Okay, if you want different status codes, then let's keep this.